packages/shared/model/src/client-server-api/index.ts

```diff
@@ -27,18 +27,18 @@ export {
 }
 
 export enum Services {
-	Example = 'example',
+	Example = 'api/example',
```

Comment:
I'm not entirely happy with this, but I wanted to scope all the rest endpoints under `/api/`, and this was the simplest and most durable way I could find.